core/dbt/events/types.py

```diff
@@ -266,7 +266,7 @@ def message(self) -> str:
 
 @dataclass
 class SystemReportReturnCode(DebugLevel, Cli, File):
-    code: int
+    returncode: int
```

Comment:
Why is this an int instead of a string?

Reply:
I think I just guessed. I'll change it to a string. Mypy should tell us once we put all the concrete codes in place and satisfy those abstract method errors.